docs/content/en/docs/core-concepts/assets/usecase-orchestrate.md_1.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnEvaluationDefinition
+metadata:
+  name: evaluate-dependencies
+  namespace: simplenode-dev
+spec:
+  objectives:
+    - keptnMetricRef:
+        name: available-cpus
+        namespace: simplenode-dev
+      evaluationTarget: ">4"
```

Comment:
Can you please add a newlines to all of the files? It will fix most of the yamllint checks